src/07-even-more-loops.js

```diff
@@ -28,7 +34,14 @@ function shoutForLoop(array) {}
  *  shoutWhileLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutWhileLoop(array) {}
+function shoutWhileLoop(array) {
+  let newArray = [];
+  while (array.length > 0){
+    //console.log (array.splice(0,1));
```

Comment:
Start thinking about removing console.logs before pushing - good habit to get into in case engineers from teams go snooping around your github